### PR TITLE
Adding pre-release license terms

### PR DESCRIPTION
**Pull request by conniey**

### CODE REVIEW

**Comment by akoeplinger** [comment]
Is there a reason for .doc instead of .md?

**Comment by conniey** [comment]
Currently talking to LCA about the file format/contents before I merge it

File: docs/LicenseTerms/Microsoft .NET Portability Analyzer Pre-release License Terms.txt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* Internet-based services, and
+* support services
+for this software, unless other terms accompany those items. If so, those terms apply.
+BY USING THE SOFTWARE, YOU ACCEPT THESE TERMS. IF YOU DO NOT ACCEPT THEM, DO NOT USE THE SOFTWARE.
+IF YOU COMPLY WITH THESE LICENSE TERMS, YOU HAVE THE PERPETUAL RIGHTS BELOW.
+1. INSTALLATION AND USE RIGHTS. 
+a. Installation and Use. You may install and use any number of copies of the software to design, develop and test your programs.  You may modify, copy, distribute or deploy any .js files contained in the software as part of your programs.
+b. Third Party Programs. The software may include third party programs that Microsoft, not the third party, licenses to you under this agreement. Notices, if any, for the third party program are included for your information only.
+2. ADDITIONAL LICENSING REQUIREMENTS AND/OR USE RIGHTS.
+a. DISTRIBUTABLE CODE.� In addition to the .js files described above, the software is comprised of Distributable Code. �Distributable Code� is code that you are permitted to distribute in programs you develop if you comply with the terms below.
```
Comment (twsouthwick) [inline docs/LicenseTerms/Microsoft .NET Portability Analyzer Pre-release License Terms.txt]: Looks like encoding issues

**Comment by twsouthwick** [comment]
LGTM - squash then merge

**Pull request merged**